src/filters.js

```diff
@@ -119,6 +119,7 @@ var filters = {
         if(str instanceof r.SafeString) {
             return str;
         }
+        if (str == null) str = '';
```

Comment:
`if(value === null || value === undefined || value === false) {`

`render('{{ foo }}', { foo: false })` should render `false` or `empty string` ?